staging/src/k8s.io/cri-api/pkg/apis/services.go

```diff
@@ -132,5 +132,5 @@ type ImageManagerService interface {
 	// RemoveImage removes the image.
 	RemoveImage(ctx context.Context, image *runtimeapi.ImageSpec) error
 	// ImageFsInfo returns information of the filesystem that is used to store images.
-	ImageFsInfo(ctx context.Context) ([]*runtimeapi.FilesystemUsage, error)
+	ImageFsInfo(ctx context.Context) (runtimeapi.ImageFsInfoResponse, error)
```

Comment:
wait was https://github.com/kubernetes/kubernetes/pull/120914/files#diff-7a3c2723dedc6b44cf183fc52f84cc4015ffbd65a998a65c3959a953cb63c945R1579 just not used before? I thought we were being purely additive here, not changing an existing api

Reply:
So from what @mikebrow and I discussed, this `services.go` is not really a component of API.  Since I don't change anything in the CRI API, I think this file is free to change.  I think the CRI API (the proto files) are what container runtime projects use.  I'm not as clear why this file is versioned in the API.  

We are adding a new API field to the CRI API but I can't really get away with not changing this as the existing code would always assume that there is only a image_filesystem.  We debated about returning multiple fields for image_filesystem and adding a label to distinguish between either image or container filesystem.  I figured that this would be more clear than having each container runtime have to label what type of filesystem this object corresponds too.

Reply:
It seems that the only component that uses this package is [critools](https://github.com/kubernetes-sigs/cri-tools/blob/master/vendor/k8s.io/cri-api/pkg/apis/services.go).  

CRI-O and containerd don't have this file in their vendored package of cri-api.